java/Makefile

```diff
@@ -453,6 +456,33 @@ verify-plume-jar-classfile-version:
 verify-plume.jar:
 	echo "TODO"
 
+###
+### build plume-core.jar file
+###
+
+# Files that depend on the jar files in java/lib and shouldn't be
+# included in plume-core
+PLUME_CORE_REMOVE = \
+	core-jar-contents/plume/ICalAvailable*.class \
+	core-jar-contents/plume/HtmlPrettyPrint*.class \
+	core-jar-contents/plume/TimeLimitProcess*.class \
+	core-jar-contents/plume/DeclarationAnnotations*.class \
+	core-jar-contents/plume/BCELUtil*.class \
+
+plume-core.jar: ${JAVA_AND_CPP_FILES} lib
+	$(MAKE) tools_jar_exists
+	$(MAKE) clean optionsdoc
```

Comment:
Calling "make clean" removes `plume.jar` which may be needed later in the build process.
Putting plume-core.jar at the ends of all lists of targets might fix this.